src/qt_gui/settings_dialog.cpp

```diff
@@ -153,6 +156,12 @@ SettingsDialog::SettingsDialog(std::span<const QString> physical_devices, QWidge
             auto checkUpdate = new CheckUpdate(true);
             checkUpdate->exec();
         });
+#else
+        // FIXME Can we do better than this?
```

Comment:
i think it depends.

im of the opinion that we should likely disable the entire update management dialog unless we strategically rethink the update channels and how that is handled.  with it as is, for a package, it would likely load the diff from nightly, but idk if thats helpful since we cant guarantee that packages would track nightly at all

Reply:
This is in the settings dialog though. I have disabled the whole update dialog and the check update from the menu.

Reply:
My goal here is to somehow hide the update section in the settings, or make it look like be disabled.